docker/README.md

```diff
@@ -0,0 +1,81 @@
+# What is Docker?
```

Comment:
```suggestion
# Webots & ROS (1) Inside Docker

A demonstration of using Docker as a development environment for ROS (1) and Webots, even though ROS (1) is deprecated.

## What is Docker?
```

`What is Docker?` is too high, the main title should be more descriptive.

Reply:
Done, thank you for the suggestion.